public/docs/ts/latest/guide/appmodule.jade

```diff
@@ -0,0 +1,136 @@
+include ../_util-fns
+
+:marked
+  An Angular Module class describes how the application parts fit together.

```

Comment:
"Module" should probably be uncapitalized, here, on the next line and throughout the text. @kwalrath, @kapunahelewong, what do you think? Maybe you want to put it in bold instead (or link to the glossary entry)?